lms/djangoapps/ccx/migrations/0004_change_ccx_coach_to_staff.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import logging
+
+from django.db import migrations
+# We're doing something awful here, but it's necessary for the greater good:
+from django.contrib.auth.models import User
```

Comment:
That is what I was referring to. In general, importing actual models into the migration is a bad idea, since it creates the assumption that the model exists in the current state of the code as it does in the current state of the database. It's better practice, in general, to use `apps.get_model`, but given that the User object is relatively static, and we're not modifying it in this migration, it matters less.

We needed to use the User object directly here, rather than through `apps.get_model` because of several methods that expect a true User object, rather than a historical object. Some even worse patching would have been necessary otherwise.